packages/core/src/amazonq/webview/ui/tabs/generator.ts

```diff
@@ -11,7 +11,7 @@ import { qChatIntroMessageForSMUS, TabTypeDataMap } from './constants'
 import { agentWalkthroughDataModel } from '../walkthrough/agent'
 import { FeatureContext } from '../../../../shared/featureConfig'
 import { RegionProfile } from '../../../../codewhisperer/models/model'
-
+import globals from '../../../../shared/extensionGlobals'
```

Comment:
this file is running in the UI but you're importing something from the node side of the extension, that's why your chat panel wont load